### PR TITLE
Akin aguda/rebranding

### PR DESCRIPTION
**Pull request by AkinAguda**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -772,10 +772,10 @@ <h5>Name goes here</h5>
         <div class="row flex-column-reverse flex-sm-row flex-lg-row">
             <div class="col-md-4 col-12">
                 <div class="footer-widget first-of-footer-widget">
-                    <img data-src="images/footer-logo.png" class="logo-sm mb-10" alt="DSC Kisii University footer logo">
+                    <img data-src="images/footer-logo.png" class="logo-sm mb-10" alt="DSC University Of Lagos footer logo">
                     <p>Copyright &copy; 2019. All Rights Reserved.</p>
                     <span>Feel free to drop us an email at </span>
-                    <a href="mailto:team@dsc.scuniversityname.org">team@dscuniversityname.org</a>
+                    <a href="mailto:team@dsc.scuniversityname.org">team@dscunilag.org</a>
```
Comment (ashinzekene) [inline index.html]: 🤔

File: learn.html [hunk 2]
```diff
@@ -156,7 +156,7 @@ <h5 class="card-title">Intro to JavaScript</h5>
                     <img src="images/footer-logo.png" class="logo-sm mb-10" alt="Image">
                     <p>Copyright &copy; 2019. All Rights Reserved.</p>
                     <span>Feel free to drop us an email at </span>
-                    <a href="mailto:team@dsc.scuniversityname.org">team@dscuniversityname.org</a>
+                    <a href="mailto:team@dsc.scuniversityname.org">team@dscunilag.org</a>
```
Comment (ashinzekene) [inline learn.html]: 🙄

File: projects.html [hunk 3]
```diff
@@ -168,7 +168,7 @@ <h5 class="card-title">Project title</h5>
                     <img src="images/footer-logo.png" class="logo-sm mb-10" alt="Image">
                     <p>Copyright &copy; 2019. All Rights Reserved.</p>
                     <span>Feel free to drop us an email at </span>
-                    <a href="mailto:team@dsc.scuniversityname.org">team@dscuniversityname.org</a>
+                    <a href="mailto:team@dsc.scuniversityname.org">team@dscunilag.org</a>
```
Comment (ashinzekene) [inline projects.html]: Probably ask Hope

**Comment by ashinzekene** [comment]
Pull in and fix merge conflicts 😄

**Comment by ashinzekene** [comment]
Add this to you comment above

Fixes #1

File: learn.html (hunk 2 above)
Comment (ashinzekene) [reply learn.html]: Use this email blessyn2hope@gmail.com

**Comment by ashinzekene** [comment]
If feels like your task is clashing with some others

**Comment by AkinAguda** [comment]
I have resolved conflicts

**Comment by ashinzekene** [comment]
Nice work, but you kinda worked on #2 which should be in a separate PR

**Comment by BenjaminAlamu** [comment]
Uncle @AkinAguda You seem to have merge conflicts

**Pull request closed without merging**